chart/templates/content-service-rolebinding.yaml

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2021 Gitpod GmbH. All rights reserved.
+# Licensed under the MIT License. See License-MIT.txt in the project root for license information.
+
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: content-service
+  labels:
+    app: {{ template "gitpod.fullname" . }}
+    component: content-service
+    kind: role-binding
+    stage: {{ .Values.installation.stage }}
+subjects:
+- kind: ServiceAccount
+  name: content-service
+roleRef:
+  kind: ClusterRole
+  name: {{ .Release.Namespace }}-ns-psp:restricted-root-user
```

Comment:
When I apply your suggested change `content-service` does not appear in `kubectl get pods` at all. Since I don't fully understand this change, please let me know whether I should revert this change or not.